docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "2"
+
+services:
+  app:
+    image: saltcorn/saltcorn
```

Comment:
See comment on e2e.yml. Here it is a little bit less clear what we want: should this docker-compose pull the image from docker hub, or build the image from source? I think it is probably best to build from source. It is my understanding that in that case we should specify the docker file (the .dev one) instead of the image but I could be wrong.